docker/pelias.json

```diff
@@ -0,0 +1,14 @@
+{
+  "logger": {
+    "level": "info"
+  },
+  "imports": {
+    "whosonfirst": {
+      "datapath": "/whosonfirst",
+      "importPostalcodes": true,
+      "importVenues": false,
+      "importPlace": "101715829",
+      "api_key": "your-api-key"
+    }
+  }
+}
```

Comment:
this example `pelias.json` seems to be tied to a specific task (101715829) without any mention of why or how to change it.